merge_reports.js

```diff
@@ -32,23 +49,27 @@ function mergeSummaries (banditSummary, gosecSummary) {
  * for reference of the 'output' object see: https://developer.github.com/v3/checks/runs/#output-object
  */
 module.exports = (banditReport, gosecReport) => {
-  let title, summary
+  let title = ''
+  let summary = ''
+  let text = ''
   let annotations = []
 
   if (banditReport.title === config.noIssuesResultTitle && banditReport.title === gosecReport.title) {
     title = config.noIssuesResultTitle
     summary = config.noIssuesResultSummary
-  } else {
```

Comment:
This should be reverted (put back in else block so there is only one return statement at the end of the function)

Reply:
I agree. I will fix it.